### PR TITLE
Copy edit Guides - Views and templates

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: guides/views.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
+Let's take a quick look at the `LayoutView` view.
 
 ```elixir
 defmodule HelloWeb.LayoutView do
   use HelloWeb, :view
 end
 ```
 
-That's simple enough. There's only one line, `use HelloWeb, :view`. This line calls the `view/0` function we just saw above. Besides allowing us to change our template root, `view/0` exercises the `__using__` macro in the `Phoenix.View` module. It also handles any module imports or aliases our application's view modules might need.
+That's simple enough. There's only one line, `use HelloWeb, :view`. This line calls the `view/0` function we just saw above. (### SAW ABOVE WHERE?####) Besides allowing us to change our template root, `view/0` exercises the `__using__` macro in the `Phoenix.View` module. It also handles any module imports or aliases our application's view modules might need.
```
Comment (eksperimental) [inline guides/views.md:23]: This is pending.
> This line calls the `view/0` function we just saw above.

File: guides/views.md [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
```
Comment (josevalim) [inline guides/views.md:23]:
```suggestion
That's simple enough. There's only one line, `use HelloWeb, :view`. This line calls the `view/0` function defined in `HelloWeb` which sets up the basic imports and configuration for our views and templates.
```

File: guides/views.md [hunk 3]
```diff
@@ -278,4 +278,12 @@ end
 
 ## Error pages
 
-Phoenix has a view called the `ErrorView` which lives in `lib/hello_web/views/error_view.ex`. The purpose of the `ErrorView` is to handle errors in a general way, from one centralized location.  Similar to the views we built in this guide, error views can return both HTML and JSON responses. See the [Custom Error Pages How-To](custom_error_pages.html) for more information.
+Phoenix has a view called the `ErrorView` which lives in `lib/hello_web/views/error_view.ex`. The purpose of `ErrorView` is to handle errors in a general way, from one centralized location.  Similar to the views we built in this guide, error views can return both HTML and JSON responses. See the [custom error pages how-to](custom_error_pages.html) for more information.
+
+
+[welcome page]: http://localhost:4000
```
Comment (josevalim) [inline guides/views.md:284]: We don't use this style anywhere else in the guides. Can you please keep the links inline? Thank you.

Comment (eksperimental) [reply guides/views.md:284]: We just did
Please see the discussion for the reasoning.
https://github.com/phoenixframework/phoenix/pull/4202#pullrequestreview-588497696

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**